### DISCUSSION TITLE
Feature request: Nested Groups

### DISCUSSION THREAD

**Discussion by brean** [open-source-labs/Svelvet#381] [Ideas]
Hi!

I like to have nested Groups so the expected behavior would be that if i move the outer group all inner groups should move with it:

```svelte
  <Group groupName={"outer"} width={500} height={350} position={{x: 0, y: 0}}>
    <Group groupName={"inner"} width={300} height={200} position={{x: 50, y: 50}}>
      <Node label="node" position={{x: 50, y: 50}} />
    </Group>
  </Group>
```

But the groups are separated from each other. When I move the inner Group it moves the node as well, the outer group does not change the position of the inner group nor the node.

Would be a nice feature to have nested Groups.

**Comment by dgrbrady** [comment]
Would love to see this added as well. I created an issue asking the same question before I saw this discussion. https://github.com/open-source-labs/Svelvet/issues/498

**Comment by brean** [comment]
I switched to [svelteflow](https://svelteflow.dev) using its [sub-flows feature](https://svelteflow.dev/learn/guides/sub-flows).
